Modix.Bot/Extensions/CommandContextExtensions.cs

```diff
@@ -55,5 +55,26 @@ async Task RemoveReactionsAndUpdateMessage(string bottomMessage)
                 await confirmationMessage.ModifyAsync(m => m.Content = mainMessage + bottomMessage);
             }
         }
+
+        public static Task<IUserMessage> ReplyWithEmbed(this ICommandContext context, string message)
```

Comment:
Don't we already have one of these, somewhere?

Reply:
Maybe? I wasn't able to find one. Though this extension method works as a drop in replacement for `ReplyAsync()` (though there's no corresponding `ModuleBase` extension so you currently have to call this through `Context.ReplyWithEmbed()`.

If I find it and it already exists I'll pull it out, but otherwise it should help us with the plan to change all replies to embeds, where applicable.